### DISCUSSION TITLE
Request: Provide Unreal as option for Built-in debug repository

### DISCUSSION THREAD

**Discussion by benpeck-eepics** [getsentry/support#40] [Support]
In a project you can add built-in debug repos such as Microsoft, Apple, Unity.

I would greatly appreciate having access to an Unreal builti-in repo that includes debug data from their major releases, eg 4.27.x, 5.0.x

Currently I have to manually upload my engine's debug files per project, which is a lot of duplication and storage usage.

Thanks, have a nice day

Ben

**Comment by bruno-garcia** [comment]
Does Epic Games provide a symbol server we can use to pull these symbols? If not, it sounds like we'd need to build our own symbol collection system and symbol server.

**Comment by benpeck-eepics** [comment]
Good question, I will inquire to Epic about this and get back to you.

**Comment by benpeck-eepics** [comment]
Epic has responded and said they do no provide a public symbol server. They
offer the symbols to be downloaded during an install of the unreal engine
from the Epic Game Launcher, but that is it.
